service/src/main/resources/templates/ehr_narrative_statement_template.mustache

```diff
@@ -0,0 +1,6 @@
+<NarrativeStatement classCode="OBS" moodCode="EVN">
+    <id root="{{narrativeStatementId}}" />
+    <statusCode code="COMPLETE" />
+    <availabilityTime value="{{availabilityTime}}" />
+    <text>{{{comment}}}</text>
```

Comment:
I think we need to use double braces {{}} here so that special characters are escaped